pkg/mkimage-raw-efi/install

```diff
@@ -8,6 +8,7 @@
 # in /bits, some will be constructed on the fly depending
 # on settings that were passed via kernel command line:
 #   eve_blackbox
+#   eve_nuke_disks
```

Comment:
Do we document these settings somewhere?

Reply:
Cannot find it in EVE repo